apps/multisig/src/components/TransactionSidesheet/TransactionSidesheetFooter.tsx

```diff
@@ -54,6 +63,36 @@ export const SignerCta: React.FC<{
   }, [isCreating, loading])
 
   const missingExecutionCallData = useMemo(() => readyToExecute && !t.callData, [readyToExecute, t.callData])
+  const firstApproval = useMemo(() => {
+    if (!t) return null
+    return !Object.values(t.approvals).find(v => v === true)
+  }, [t])
+
+  const connectedAccountHasEnoughBalance: boolean = useMemo(() => {
+    if (asDraft || existentialDepositLoadable.state === 'loading') return true
+
+    let txCost = BigInt(fee?.amount.toString() ?? 0) + BigInt(existentialDepositLoadable.contents.amount.toString())
+    if (firstApproval) {
+      txCost += BigInt(multisigDepositTotal.contents.amount?.toString() ?? 0)
+    }
+
+    const [connectedWalletBal] =
+      balances?.find(({ address }) => {
+        const parsedAddress = Address.fromSs58(address)
+        return parsedAddress && !!user && parsedAddress.isEqual(user.injected.address)
+      }) || []
```

Comment:
Since balances can contain cross chain balances, we should make sure we're checking against the user's balance on the right chain:

```suggestion
      balances?.find(({ address, chainId }) => {
        const parsedAddress = Address.fromSs58(address)
        return parsedAddress && !!user && parsedAddress.isEqual(user.injected.address) && chainId === t.multisig.chain.squidIds.chainData
      }) || []
```

Make sure to also handle useMemo's dependencies!